pkg/api/meta/meta.go

```diff
@@ -321,16 +321,18 @@ func (resourceAccessor) SetResourceVersion(obj runtime.Object, version string) e
 // genericAccessor contains pointers to strings that can modify an arbitrary
 // struct and implements the Accessor interface.
 type genericAccessor struct {
-	namespace       *string
-	name            *string
-	generateName    *string
-	uid             *types.UID
-	apiVersion      *string
-	kind            *string
-	resourceVersion *string
-	selfLink        *string
-	labels          *map[string]string
-	annotations     *map[string]string
+	namespace         *string
```

Comment:
I'm not explicitly calling genericAccessor anywhere. This is a fallback in case of non-internal object (for internal objects we will go to a different path).
So since in huge majority of cases it will be an internal object, we will not get here.
So this is WAI in my opinion (even though I'm changing it here).